docs/src/components/DelayedRenderer.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
+import { Loader, Segment, LoaderProps } from '@stardust-ui/react'
+
+type DelayedRendererProps = Partial<{
+  children: JSX.Element
+  placeholder: JSX.Element | LoaderProps['size']
+  visible: boolean
+}>
```

Comment:
Do we need really need this complication? I don't see any usage of `placeholder` as element.

I tried to rewrite this file and it looks more simple:

```tsx
import * as React from 'react'
import { Loader, Segment, LoaderProps } from '@stardust-ui/react'

type DelayedRendererProps = {
  placeholder: LoaderProps['size']
  visible: boolean
}

const paddings: Record<LoaderProps['size'], string> = {
  smallest: '0px',
  smaller: '5px',
  small: '15px',
  medium: '25px',
  large: '40px',
  larger: '65px',
  largest: '100px',
}

const DelayedRenderer: React.FunctionComponent<DelayedRendererProps> = props => {
  const { children, placeholder, visible } = props

  return visible ? <>{children}</> : (
    <Segment variables={{ padding: paddings[placeholder] }}>
      <Loader size={placeholder}/>
    </Segment>
  )
}

export default DelayedRenderer
```

Reply:
We don't need it for sure, agreed that if users of this component need custom placeholder we can implement something on demand and that probably won't happen.

Then I'll probably change the name of `placeholder` prop to `size` or `placeholderSize`. Which one are you voting for @layershifter ?

Also,  is `DelayedRenderer` a good enough name?

Reply:
> Then I'll probably change the name of placeholder prop to size 

👍 

> Also, is DelayedRenderer a good enough name?

Not, sure because there are no any `delay` in that component

Reply:
What about `ExamplePlaceholder`?

Reply:
applied suggestions